### DISCUSSION TITLE
Create rule with input data (json)

### DISCUSSION THREAD

**Discussion by elohim85** [mockoon/mockoon#406] [Help] [answered]
Hi All,

It is possible, in the mockery, to create a rule that validates the following: if the request I am going to send from Postman matches this data from the rule, then it must return a code 200 with the response. eg:

Request:
```
{
  "source": {
    "name": "Pedro",
    "lastname": "Yepes",
    "age": "35"
   }
}
```

Rules
name= Pedro
lastname= Yepes
age =35

Thank you very much for your help!

**Comment by 255kb** [comment, marked as answer]
Hi @elohim85 
Sure you can. 
Here is the setup: One 401 default response (first one in the list) with no rules, one 200 with 3 "AND" rules.
![image](https://user-images.githubusercontent.com/7489814/104817092-5a785c80-581f-11eb-85dd-c25683ff9e32.png)
![image](https://user-images.githubusercontent.com/7489814/104817094-5f3d1080-581f-11eb-838f-59d8aee8d1a9.png)

Entering request *must* have a `application/json` Content-Type header

**Reply by elohim85** [reply]
Very thanks for your help, my mocks are working.

Regards

**Comment by elohim85** [comment]
Hi again,

If I have 2 arrays in my request:  

```
{
    "partyadddata": {
        "attribute": [{
                "name": "servtype",
                "value": "99"
            }, {
                "name": "authcode",
                "value": "AAA"
            }
        ]
    },
    "source": {
        "name": "Pedro",
        "lastname": "Yepes",
        "age": "35"
    }
}
```

The rule is OK if I set it up like this:

![image](https://user-images.githubusercontent.com/32969789/105738399-9cc72b00-5efc-11eb-8f7f-15815b2ab32f.png)

Thank you very much for your help!

**Reply by 255kb** [reply]
I just tested your setup and I got the 200 answer. It seems to be ok.
